server/handler/groups/group.go

```diff
@@ -0,0 +1,86 @@
+package groups
+
+import (
+	"net/http"
+
+	"docktor/server/dao"
+	"docktor/server/types"
+
+	"github.com/labstack/echo"
+	log "github.com/sirupsen/logrus"
+)
+
+// getAll find all groups
+func getAll(c echo.Context) error {
+	groups, err := dao.GetGroups()
+	if err != nil {
+		log.WithFields(log.Fields{
+			"error": err,
+		}).Error("Error when retrieving groups")
+		return c.JSON(http.StatusBadRequest, err.Error())
+	}
+	daemons, err := dao.GetDaemons()
+	if err != nil {
+		log.WithFields(log.Fields{
+			"error": err,
+		}).Error("Error when retrieving daemons")
+		return c.JSON(http.StatusBadRequest, err.Error())
+	}
+	for key, group := range groups {
```

Comment:
Do it in dao

Reply:
remove duplication https://sonarcloud.io/component_measures?id=thecampagnards_docktor&metric=new_duplicated_lines_density&pullRequest=27&selected=thecampagnards_docktor%3Aserver%2Futils%2Fcompose.go

Reply:
add unit test for admin

Reply:
Tu parles tout seul mec ?

Reply:
@SmalkiGroove yep xD